tools/packager/afu.py

```diff
@@ -92,8 +87,8 @@ def validate(self, packaging=False):
         if self.afu_json == {}:
             return False
         try:
-            jsonschema.validate(self.afu_json, afu_schema)
-        except jsonschema.exceptions.ValidationError as ve:
+            validators.validate(self.afu_json, afu_schema)
```

Comment:
`jsonschema.validate()` isn't really even more convenient than `validators.validate()` and by my reading does the same thing.